### PR TITLE
Fix null ent errors

### PR DESCRIPTION
**Pull request by wrefgtzweve**

### CODE REVIEW

File: lua/entities/rail_shell/init.lua [hunk 1]
```diff
@@ -38,7 +40,7 @@ function ENT:Think()
 
     if self.AirburstTime < CurTime() then
         local owner = IsValid( self:GetOwner() ) and self:GetOwner()
-        local inflictor = owner or self.Turret
+        local inflictor = IsValid( owner ) and owner or IsValid( self.Turret ) and self.Turret or self
```
Comment (loganlearner) [inline lua/entities/rail_shell/init.lua:43]: Perhaps some parenthesis to make this a bit more readable?

**Review by loganlearner — CHANGES_REQUESTED** [review]

File: lua/entities/rail_shell/init.lua (hunk 1 above)
Comment (wrefgtzweve) [reply lua/entities/rail_shell/init.lua:43]: not sure how you'd do that, it will get ugly even with parenthesis as far as i know

Comment (loganlearner) [reply lua/entities/rail_shell/init.lua:43]:
```suggestion
        local inflictor = IsValid( owner ) and owner or ( IsValid( self.Turret ) and self.Turret or self )

```

Comment (wrefgtzweve) [reply lua/entities/rail_shell/init.lua:43]: returns nil

Comment (wrefgtzweve) [reply lua/entities/rail_shell/init.lua:43]: oh wait no, but the self is still in a weird spot there

**Pull request merged**